contracts/libraries/LibArenaStorage.sol

```diff
@@ -52,7 +52,7 @@ struct ArenaConstants {
     uint256 CLAIM_VICTORY_ENERGY_PERCENT;
     uint256 START_TIME;
     bool NO_ADMIN;
-    uint256 [] INIT_PLANET_IDS; // TODO: Confirm this won't mess up Diamond storage
+    bytes32 [] INIT_PLANET_HASHES; // TODO: Confirm this won't mess up Diamond storage
```

Comment:
did you do this?

Reply:
https://dev.to/mudgen/how-diamond-storage-works-90e

https://docs.soliditylang.org/en/develop/internals/layout_in_storage.html?highlight=bytes#mappings-and-dynamic-arrays

2 reasons why I think this is ok:
1) Dynamic arrays are not stored contiguously, the location for them is at `keccak256(p)` where `p` is the storage slot of `INIT_PLANET_HASHES`. So putting another value below won't mess up diamond storage.
2) This storage change will only be relevant for new arenas that are created -> we don't care if the base Diamond gets its constants overwritten as long as new Diamonds have the correct storage pattern.